### PR TITLE
Add function_deploy_timeout and post_deploy_cleanup

### PR DESCRIPTION
**Pull request by haakonvt**
Adds the two new action parameters to the template.
+ spellcheck
+ update name on help-forge channel 😅

### CODE REVIEW

File: .github/workflows/deploy-push-master.yaml [hunk 1]
```diff
@@ -26,7 +27,7 @@ jobs:
       - name: Resolve secrets
         shell: bash
         env:
-          BRANCH: ${{ steps.extract_name.outputs.under_name }}
+          BRANCH: master # ${{ steps.extract_name.outputs.under_name }}
```
Comment (haakonvt) [inline .github/workflows/deploy-push-master.yaml:30]:
```suggestion
          BRANCH: ${{ steps.extract_name.outputs.under_name }}
```

File: .github/workflows/deploy-push-master.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
+      - add-new-action-params
```
Comment (haakonvt) [inline .github/workflows/deploy-push-master.yaml:6]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -135,4 +135,4 @@ For more details on how to set up your workflows to deploy to Cognite Functions,
 
 # I have a question
 
-There will be questions. Crowdsourcing is required. Use issues, or if you are an internal Cognite employee, you can also hit us up on slack on #help-forge
+There will be questions. Crowdsourcing is required. Use issues, or if you are an internal Cognite employee, you can also hit us up on slack on #help-inso-tech-community
```
Comment (haakonvt) [inline README.md:138]: # speed

File: example_function2/function_config.yaml [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# Note: Parameters can be hardcoded directly in 'deploy-push.yaml'
+# if you do not need to vary them between your functions, for
+# example 'cdf_project' or 'cdf_cluster'.
 cdf_project: forge-sandbox
 cdf_cluster: greenfield  # or api, westeurope-1, etc.
```
Comment (ks93) [inline example_function2/function_config.yaml:5]: Should default be greenfield?

**Review by ks93 — APPROVED** [review]
One question, otherwise LGTM!

File: example_function2/function_config.yaml (hunk 4 above)
Comment (haakonvt) [reply example_function2/function_config.yaml:5]: No 😜 This repo deploys on merges to master, so the parameters are real!

**Pull request merged**